tuhi/export.py

```diff
@@ -82,27 +82,44 @@ class JsonSvg(ImageExportBase):
     _base_pen_width = 0.4
     _pen_pressure_width_factor = 0.2
 
+    # Change this value down to reduce size, change it up to improve accuracy. measured in px
+    width_precision = 10
```

Comment:
```suggestion
    _width_precision = 10
```
for consistency with the other fields

Reply:
OK, I didn't hide it because it seemed like a useful value to pass in at some point. But conventions are conventions ;-D

Reply:
well, this isn't a real API, it's just a class we use internally. We could export it as `@property` but let's do that when we have actual use-cases, no need to go over the top in the first version.

Reply:
Should be done.

Reply:
I think you forgot to push :)